Add small explanation on PRs to the TestGrid. (#2626)
kyma-project/test-infra: docs/prow/prow-k8s-testgrid.md

TestGrid configuration is stored inside the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/tree/master/config/testgrids/kyma) repository in the `/config/testgrids/kyma/` directory.

TestGrid is automatically updated by Prow using the tool called [transfigure.sh](https://github.com/kubernetes/test-infra/tree/master/testgrid/cmd/transfigure).

> It creates pull request each time the TestGrid configuration file in the kubernetes/test-infra gets changed. The changes will appear when the owner of the `/config/testgrids/kyma/` folder approves the changes. For more information see the `transfigure` *README.md* file.

Reviewer: ```suggestion
It creates a pull request each time the TestGrid configuration file in the `kubernetes/test-infra` is changed. The changes appear when the owner of the `/config/testgrids/kyma/` folder approves them. For more information, see the `transfigure` *README.md* file.
```
